src/cpp/session/SessionClientInit.cpp

```diff
@@ -380,6 +380,8 @@ void handleClientInit(const boost::function<void()>& initFunction,
 
    sessionInfo["session_id"] = module_context::activeSession().id();
 
+   sessionInfo["drivers_support_licensing"] = false;
```

Comment:
Presumably this will need to have a different value in Pro. We generally try to keep the body of this method identical; you could use something like `!allowOverlay()` to toggle the option:

https://github.com/rstudio/rstudio/blob/54cd3abcfc58837b433464c793fe9b03a87f0bb4/src/cpp/session/SessionOptionsOverlay.cpp#L37-L40

Alternately, you could add a specific overlay option/method.